scripts/local/dev.ts

```diff
@@ -40,6 +40,8 @@ void async function () {
     process.env.PROJECT = process.env.PROJECT || 'casimir'
     process.env.STAGE = process.env.STAGE || 'dev'
     process.env.PUBLIC_STAGE = process.env.STAGE // Pass stage to client apps
+    process.env.CRYPTO_COMPARE_API_KEY = await getSecret('casimir-crypto-compare-api-key')
+    console.log('process.env.CRYPTO_COMPARE_API_KEY :>> ', process.env.CRYPTO_COMPARE_API_KEY)
```

Comment:
Remove.

Reply:
Will also need to add `PUBLIC_` prefix if this is being used in the web app. And we need to set that is the prefix in the Vue config if it is not set.